vespalib/src/vespa/vespalib/util/threadstackexecutorbase.cpp

```diff
@@ -72,22 +72,22 @@ ThreadStackExecutorBase::unblock_threads(const MonitorGuard &)
 void
 ThreadStackExecutorBase::assignTask(TaggedTask task, Worker &worker)
 {
-    MonitorGuard monitor(worker.monitor);
+    unique_lock monitor(worker.lock);
```

Comment:
this should probably also be called guard

Reply:
#14896